sdk/src/main/kotlin/com/processout/sdk/ui/web/customtab/CustomTabAuthorizationViewModel.kt

```diff
@@ -59,8 +61,19 @@ internal class CustomTabAuthorizationViewModel(
         val returnUri = intent.data
         if (returnUri != null) {
             timeoutHandler.removeCallbacks(cancellationRunnable)
-            POLogger.info("Custom Chrome Tabs has been redirected to return URL: %s", returnUri)
-            savedState[KEY_SAVED_STATE] = Success(returnUri)
+            if (returnUri.toString().startsWith(configuration.returnUri.toString())) {
```

Comment:
I believe that following combination should result in success, but I'm not sure it will. Could you please check?
```
configuration.returnUri: test://return?
returnUri: "test://return"
```

Reply:
Good point, fixed for `CustomTabAuthorizationViewModel` and `ProcessOutWebView`